atomic.conf

```diff
@@ -3,3 +3,4 @@
 default_scanner: openscap
 default_docker: docker
 
+# default_storage: ostree
```

Comment:
Why not make this the default?  What other choices do I have?  Should we list options in the config file?